source/more-info/unsupported/home_assistant_core_version.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: "Home Assistant Core version"
```

Comment:
Home Assistant Core version is a very generic title. Can we add the symptom in the title? Or is there a reason to just mention that it's something about the Core version?

This page seems like a troubleshooter to me. 
We could use the Symptom, Description (or Cause), Resolution structure here.